### DISCUSSION TITLE
Release 0.9.9

### DISCUSSION THREAD

**Discussion by Ximi1970** [Ximi1970/systray-x#188] [Announcements]
The Windows packages will install the addon and companion app system wide, see 'Assets' below.

Windows 10 / 11 64-bit
Windows 10 / 11 32-bit

For Linux, SysTray-X is available in the repositories at the OBS and can be found here:

https://download.opensuse.org/repositories/home:/Ximi1970:/Mozilla:/Add-ons

You can find the install procedures in the README.md file at https://github.com/Ximi1970/systray-x.

Packages for the following distributions are available at the OBS:

openSuSE Leap 15.4
openSuSE Leap 15.5
openSuSE Leap 15.6
openSuSE Slowroll x86_64 (Qt6)
openSuSE Tumbleweed i586 (Qt6)
openSuSE Tumbleweed x86_64 (Qt6)
Kubuntu / Xubuntu / Ubuntu 18.04 LTS
Kubuntu / Xubuntu / Ubuntu 20.04 LTS
Kubuntu / Xubuntu / Ubuntu 22.04 LTS
Kubuntu / Xubuntu / Ubuntu 23.04
Kubuntu / Xubuntu / Ubuntu 23.10
Kubuntu / Xubuntu / Ubuntu 24.04 LTS
Debian 10
Debian 11
Debian 12
Debian Unstable
Fedora 37
Fedora 38
Fedora 39
Fedora 40 (Qt6)
Fedora Rawhide (Qt6)
Manjaro, Arch (Qt6)

<hr /><em>This discussion was created from the release <a href='https://github.com/Ximi1970/systray-x/releases/tag/0.9.9'>Release 0.9.9</a>.</em>

**Comment by dca00** [comment]
Hi!
Is there going to be Fedora 41 repo? Or should users of F41 install from the F40 repo?
Been using FireTray with TB 60, but in F41 it stopped working, so looking for an alternative.

**Comment by Ximi1970** [comment]
Added the Fedora 41 build. Newest release is 0.9.10.
